.github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml

```diff
@@ -0,0 +1,24 @@
+name: Feature Request
+description: File a feature request
+title: "[Feature]: "
+labels: ["enchancement"]
+body:
+  - type: textarea
+    id: detail
+    attributes:
+      label: Detail
+      description: Please explain the feature in details.
+    validations:
+      required: true
+  - type: textarea
+    id: reason
+    attributes:
+      label: Reason
+      description: Reason for adding this feature.
+    validations:
+      required: true
+  - type: textarea
+    id: pseudo-code
+    attributes:
+      label: Pseudo Code
+      description: If possible, Please include a pseudo code on how the feature will work.
```

Comment:
Give an example of how you imagine this feature would be used, using pseudo-code.